lib/term_statistics.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+class TermStatistics
+  # TODO: reduce number of arguments
+  def initialize(term:, org_lookup:, people:)
```

Comment:
I'm not sure what we want to do generally about documentation, but I think we definitely need to do _something_ to explain what these arguments are. (And I strongly suspect that doing so would reveal `org_lookup`, at least, isn't really going to be a suitable thing to be passing around)